scripts/portfolio.js

```diff
@@ -1,28 +1,22 @@
-var projects = [];  // i don't think i need this array here
+var githubRepos = [];
 
 function Project (repos) {
   // constructor function for github repos to be displayed
   this.title = repos.title;
   this.updatedOn = repos.updatedOn;
-  this.body = repos.body;
+  this.course = repos.course;
 }
```

Comment:
How can you refactor this object constructor so that you don't have to list each property?